### DISCUSSION TITLE
请教一个问题，cargo build 打包成exe之后，双击执行这个exe，会出现一个cmd框，请问应该执行哪个编译选项，感谢解答

### DISCUSSION THREAD

**Discussion by FiveKm** [syf20020816/SurrealismUI#31] [Q&A] [answered]

**Comment by syf20020816** [comment]
#![windows_subsystem = "windows"]

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: FiveKm ***@***.***>
Sent: Tuesday, April 2, 2024 3:15:28 PM
To: Surrealism-All/SurrealismUI ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Surrealism-All/SurrealismUI] 请教一个问题，cargo build 打包成exe之后，双击执行这个exe，会出现一个cmd框，请问应该执行哪个编译选项，感谢解答 (Issue #30)


―

**Comment by syf20020816** [comment, marked as answer]
I am afraid you may not understand, the following is detail:

you can add `#![windows_subsystem = "{{your system}}"]`  in main.rs

such as:

```
#![windows_subsystem = "windows")]

fn main() {
    // other code ...
}
```

**Comment by syf20020816** [comment]
you can get other info from : [slint issue #3235](https://github.com/slint-ui/slint/issues/3235) or [slint issue #1190](https://github.com/slint-ui/slint/issues/1190)
